pages/common/vim.md

```diff
@@ -1,32 +1,36 @@
 # vim
 
 > Vi IMproved, a programmer's text editor, providing several modes for different kinds of text manipulation.
-> Pressing `i` enters edit mode; the normal mode (accessed via `<Esc>`) doesn't allow regular text editing.
+> Pressing `i` enters edit mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
 
 - Open a file:
 
 `vim {{file}}`
 
 - Enter text editing mode (insert mode):
 
-`<Esc> i`
+`<Esc>i`
 
 - Copy ("yank") or cut ("delete") the current line (paste it with `P`):
 
-`<Esc> {{yy|dd}}`
+`<Esc>{{yy|dd}}`
 
 - Undo the last operation:
 
-`<Esc> u`
+`<Esc>u`
 
-- Search for a pattern in the file (press `n` to go to the next result):
+- Search for a pattern in the file:
 
-`<Esc> /{{search_pattern}} <Enter>`
+`<Esc>/{{search_pattern}}<Enter>   then   n (next), N (previous)`
 
-- Perform a regex substitution in the whole file (from the start, `1`, to the end, `$`):
+- Perform a regex substitution in the whole file:
 
-`<Esc> :1,$s/{{pattern}}/{{replacement}}/g <Enter>`
+`<Esc>:%s/{{pattern}}/{{replacement}}/g<Enter>`
```

Comment:
Not sure this was a "catch" 😄 I deliberately used the non-shorthand notation, in order to introduce the concept of ranges preceding commands. I mentioned this decision explicitly in #1105:

> I combined copy/cut/paste into a single command, since it was important to allow space for a slightly more advanced command, (the find/replace example), which **introduces some important vim concepts, such as command application ranges.**

Although `%` is shorter, it is so by mere two characters, and obscures this meaning and instead introduces a single-purpose symbol that doesn't allow this example to be built upon, which is one of the [goals of TLDR-pages](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines):

> The goal is to allow people to *understand* the syntax of a command, not just *memorize* it.

So IMO we should both retain the `1,$` range, as well as the parenthetical note in the command's description.